surfsense_backend/app/agents/researcher/nodes.py

```
@@ -172,20 +172,41 @@ async def fetch_documents_by_ids(
                     elif channel_id:
                         url = f"https://discord.com/channels/@me/{channel_id}"
                     else:
                         url = ""
-                        
+
+                elif doc_type == "JIRA_CONNECTOR":
+                    # Extract Jira-specific metadata
+                    issue_key = metadata.get('issue_key', 'Unknown Issue')
+                    issue_title = metadata.get('issue_title', 'Untitled Issue')
+                    status = metadata.get('status', '')
+                    priority = metadata.get('priority', '')
```

Ruff (0.12.2)

Local variable `priority` is assigned to but never used

Remove assignment to unused variable `priority`

(F841)